tests/releaseNotes_Test.py

```diff
@@ -27,5 +28,28 @@ def test_givenSimpleInputPrintVersionBlockGetsCalled(self):
         releaseNotes.generateReleaseNotesByPromotedVersions()
         writer.printVersionBlock.assert_called_once_with({}, "1.0.1.2", "N/A", ["TCKT-1"])
 
+    def test_givenContentTemplate_ProducesCorrectResponse(self):
+
+        conf = {"ContentTemplateFile" : "dummy.txt" }
+        promotedVersions = {}
+
+        ticketProvider = Mock()
+        ticketProvider.extractTicketsFromMessage = MagicMock(return_value=["TCKT-1"])
+        ticketProvider.getTicketInfo = MagicMock(return_value={ "issue_type_icon" : "None", "html_url" : None,
+                                                                "ticket" : "TCKT-1",   "title" : "TCKT-1 Summary",
+                                                                "embedded_link" : {} })
+
+        writer = HtmlWriter(ticketProvider, "abc ${content} xyz")
```

Comment:
I'm still curious how this template that is shares across writers could work. 
Html footer and header will look drastically different from Markdown (or any other language for that matter).

That's why I initially thought that template could be something that is writer specific. How do you imagine that ?